test/formula.jl

```diff
@@ -314,27 +314,30 @@ module TestFormula
     ## @test mm.model == [ones(4) x1 x3 x2 x1.*x2 x3.*x2]
 
     ## Interactions between three PDA columns
-    df = DataFrame(y=1:27,
-                   x1 = PooledDataArray(vec([x for x in 1:3, y in 4:6, z in 7:9])),
-                   x2 = PooledDataArray(vec([y for x in 1:3, y in 4:6, z in 7:9])),
-                   x3 = PooledDataArray(vec([z for x in 1:3, y in 4:6, z in 7:9])))
-    f = y ~ x1 & x2 & x3
-    mf = ModelFrame(f, df)
-    @test coefnames(mf)[2:end] ==
-        vec([string("x1 - ", x, " & x2 - ", y, " & x3 - ", z) for
-             x in 2:3,
-             y in 5:6,
-             z in 8:9])
+    ## 
+    ## FAILS: behavior is wrong when no lower-order terms (1+x1+x2+x1&x2...)
```

Comment:
Indeed, if it's incorrect, remove it. Though double check with e.g. R that this is the standard behavior.